fuzon-http/README.md

```diff
@@ -0,0 +1,23 @@
+# fuzon-http
+
+This is a web-server to deploy fuzon as a web-service.
+All ontologies are loaded once on server startup, and the indices are kept in memory.
+
+## Configuration
+
+The server takes a configuration file as input to determine what ontologies to load, and which collections to load them into. Collections are individual matchers which can be queried independently.
+
+## Installation
+
+```shell
+cd fuzon-http
+cargo build --release
+../target/release/fuzon-http --config config/example.json
```

Comment:
**issue(minor, nitpick)**: either we rename the section "installation and run" or we move L15 to a separate code chunk and define it "run"

Reply:
:+1: fixed in 2c79e4f